index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Border Animation</title>
+</head>
+<body>
+    <div class="frem">
```

Comment:
Please use some meaningful name for class

Reply:
surely I have noted your suggestion I will implement it from now on.